executor/src/witgen/machines/fixed_lookup_machine.rs

```diff
@@ -197,11 +236,31 @@ impl<'a, T: FieldElement> FixedLookup<'a, T> {
                 .then_some((i.id, i))
             })
             .collect();
+
+        let degree = fixed_data
+            .fixed_cols
+            .values()
+            .map(|col| col.values_max_size().len())
+            .max()
+            .unwrap_or(0) as u64;
```

Comment:
```suggestion
            // No fixed column, so this FixedLookup instance does nothing
            .unwrap_or(0) as u64;
```

This wasn't obvious to me at first glance, think a comment helps :)